HyperV/scripts/create-environment.ps1

```diff
@@ -134,6 +134,9 @@ if ($buildFor -eq "openstack/neutron"){
     }
     Get-ChildItem $buildDir
     ExecRetry {
+        if (($branchName.ToLower().CompareTo($('stable/mitaka').ToLower()) -eq 0)) {
+            GitClonePull "$buildDir\networking-hyperv" "https://git.openstack.org/openstack/networking-hyperv.git" "master"
+        }
         GitClonePull "$buildDir\networking-hyperv" "https://git.openstack.org/openstack/networking-hyperv.git" $branchName
```

Comment:
Since this is not in "else" it will still try to do another gitclonepull and fail, right?